lib/Engine.php

```diff
@@ -109,36 +109,36 @@ public function generatePassword() {
 	 */
 	public function verifyPassword($password, $uid = null) {
 		if ($this->yes('spv_min_chars_checked')) {
-			$val = $this->configValues['spv_min_chars_value'];
+			$val = (int) $this->configValues['spv_min_chars_value'];
```

Comment:
This was nasty to find. The unit tests all pass, because they send actual numbers to ``verify($password, $val)``

https://github.com/owncloud/core/blob/master/lib/private/L10N/L10N.php#L103
Does an exact comparison of the count with ``1`` - so if you pass down a variable that is the string "1" then you end up being returned the plural translation anyway.
Perhaps that code in core could be more friendly and if some Wally has passed in "1" then send them back the singular translation.